bids2openminds/report.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+
+def create_report(dataset, dataset_version, collection, dataset_description, input_path):
+    print(f"Dataset title : {dataset.full_name}")
```

Comment:
you cannot just print the individual elements, but have to write/collect them first into an object that is then printed by executing the function (with option to set to quite) and presented as an output of that function in addition to the collection.

Reply:
```
Conversion Report
=================

The following elements were converted:
--------------------------------------
+ number of converted subjects: XX
+ number of state for each subject: XX-YY
+ number of files: XX

The following elements were not converted:
------------------------------------------
+ derivative data are ignored for now
+ the following suffixes were not translated to techniques yet: [x,y,z]
```

Reply:
(this is an example if you decide for a markdown text file, keep the first version simple we can always add more information later on)